cocli/data/templates/comid-cca-realm-refval.json

```diff
@@ -0,0 +1,70 @@
+{
+    "lang": "en-GB",
+    "tag-identity": {
+        "id": "43BBE37F-2E61-4B33-AED3-53CFF1428B16",
+        "version": 0
+    },
+    "entities": [
+        {
+            "name": "Workload Client Ltd.",
+            "regid": "https://workloadclient.example",
+            "roles": [
+                "tagCreator",
+                "creator",
+                "maintainer"
+            ]
+        }
+    ],
+    "triples": {
+        "reference-values": [
+            {
+                "environment": {
+                    "class": {
+                        "id": {
+                            "type": "uuid",
+                            "value": "CD1F0E55-26F9-460D-B9D8-F7FDE171787C"
```

Comment:
It'd be handy to have a stable identifier for a given workload, but that's not the case in CCA.  I think we should use the RIM value to identify this (ephemeral) environment.

Reply:
> We cannot request supply chain to ask for RIM to be the Identifier.

why not?

> It is perfectly fine to tag a `UUID` for the Stable Identifier. This is more practical!

How is it more practical to have to invent (and keep track of) an identifier that has no real linkage to the reference value data supplied?

Reply:
Supply Chain is free to choose any real identifier they want for their Workload.
The Class ID is extensible! 

This model proposes : UUID as a Workload Identifier. 

This ID will be relayed in the EAR(Veraison Extension) when RIM and REM Matches to `RP`,

In case two Workloads Have Identical RIM's we can then use the UUID to mention both of them in EAR.

Reply:
> Supply Chain is free to choose any real identifier they want for their Workload.
> The Class ID is extensible!
>
> This model proposes : UUID as a Workload Identifier.

I appreciate that.  My point is we are making things more complex than they need to be, without a good reason.

> This ID will be relayed in the EAR(Veraison Extension) when RIM and REM Matches to RP,

not sure, what is RP?

> In case two Workloads Have Identical RIM's we can then use the UUID to mention both of them in EAR.

I don't understand what "mention both of them in EAR" means.

Reply:
`RP` Here refers to Relying Party!
Mention both of them in EAT Attestation Results!

Also there is no complexity in tagging UUID as a Workload Identity. It is just an example Identifier to uniquely identify a Workload! 

Realm specs talks about GUID as one of the identifier for a Workload. Going by that logic: UUID makes perfect sense to me!

Reply:
> Also there is no complexity in tagging UUID as a Workload Identity. It is just an example Identifier to uniquely identify a Workload!

I argue there *is* extra complexity because now the roles need to coordinate among themselves to keep track of yet another identifier which has no concrete relationship whatsoever with the workload.
 
> Realm specs talks about GUID as one of the identifier for a Workload.
> Going by that logic: UUID makes perfect sense to me!

Can you point me to the document where it is suggested to identify a given workload class using GUIDs?

Reply:
> > Also there is no complexity in tagging UUID as a Workload Identity. It is just an example Identifier to uniquely identify a Workload!
> 
> I argue there _is_ extra complexity because now the roles need to coordinate among themselves to keep track of yet another identifier which has no concrete relationship whatsoever with the workload.

Can you please explain what are the _Roles_ involved here: If  I understand correctly, there is ONLY one Supply Chain Entity the Work load Creator here!
> 
> > Realm specs talks about GUID as one of the identifier for a Workload.
> > Going by that logic: UUID makes perfect sense to me!
> 
> Can you point me to the document where it is suggested to identify a given workload class using GUIDs?

Reply:
> This ID will be relayed in the EAR(Veraison Extension) when RIM and REM Matches to `RP`,

If that kind of payload identification is needed (on top of what is already provided by the trustworthiness vector), the relying party is going to want an endorsed value that provides information about the SW run by the workload rather than an opaque identifier.

Reply:
> > I argue there _is_ extra complexity because now the roles need to coordinate among themselves to keep track of yet another identifier which has no concrete relationship whatsoever with the workload.
> 
> Can you please explain what are the _Roles_ involved here: If I understand correctly, there is ONLY one Supply Chain Entity the Work load Creator here!

RP and RVP.